test/add-service-index.test.js

```diff
@@ -19,6 +19,7 @@ test('Adds an index file to a service directory', function (t) {
       var index = fs.readFileSync(indexFile).toString()
       var template = fs.readFileSync(templateIndex).toString()
 
+      t.ok(index.includes('<title>{{service}} screenshots</title>'), 'uses the service name in the title')
```

Comment:
This test isn't really doing much. We should probably make a new suite that tests all the dynamic parts of the template get rendered as expected once data has been run through it.